docs/developers/sdk.md

```diff
@@ -524,177 +537,284 @@ Transactions received by the Coordinator will be stored in its transaction pool
+
+```js
+  // set amount to transfer
+  const amountTransferAuth = hermez.HermezCompressedAmount.compressAmount(hermez.Utils.getTokenAmountBigInt('1', 18));
+  // generate L2 transaction
+  const l2AuthTxTransfer = {
+    from: infoAccountSender.accountIndex,
+    to: hermezWallet3.hermezEthereumAddress
+    amount: amountTransferAuth,
+    fee: state.recommendedFee.createAccount / usdTokenExchangeRate;
+  };
+  const accountAuthTransferResponse = await hermez.Tx.generateAndSendL2Tx(l2AuthTxTransfer, hermezWallet, infoAccountSender.token);
+  console.log(accountAuthTransferResponse);
+ 
+>>>>>
+{
+  status: 200,
+  id: '0x025398af5b69f132d8c2c5b7b225df1436baf7d1774a6b017a233bf273b4675c8f',
+  nonce: 0
+}
+
+```
+
+After the transfer has been forged, we can check Mary's account on Hermez
+```js
+    // get receiver account information
+    const infoAccountAuth = (await hermez.CoordinatorAPI.getAccounts(hermezWallet3.hermezEthereumAddress, [tokenERC20.id]))
+      .accounts[0];
+    console.log(infoAccountAuth);
+
+>>>>>
+
+{
+  accountIndex: 'hez:ETH:265',
+  balance: '1000000000000000',
+  bjj: 'hez:ct0ml6FjdUN6uGUHZ70qOq5-58cZ19SJDeldMH021oOk',
+  hezEthereumAddress: 'hez:0xd3B6DcfCA7Eb3207905Be27Ddfa69453625ffbf9',
+  itemId: 10,
   nonce: 0,
   token: {
-    USD: null,
+    USD: 1795.94,
     decimals: 18,
-    ethereumAddress: '0x2e9f55f7266d8c7e07d359daba0e743e331b7a1a',
-    ethereumBlockNum: 61,
-    fiatUpdate: null,
-    id: 3,
-    itemId: 4,
-    name: 'ERC20_2',
-    symbol: '20_2'
+    ethereumAddress: '0x0000000000000000000000000000000000000000',
+    ethereumBlockNum: 0,
+    fiatUpdate: '2021-03-16T14:56:57.460862Z',
+    id: 0,
+    itemId: 1,
+    name: 'Ether',
+    symbol: 'ETH'
   }
 }
-
 ```
-### Force Exit
 
-This is the L1 equivalent of an Exit. With this option, the smart contract forces Coordinators to pick up these transactions before they pick up L2 transactions. Meaning that these transactions will always be picked up.
+## Create Internal Accounts
+Until now we have seen that accounts have a Ethereum address and a Baby JubJub key. This is the case for normal accounts. However, there is a second type of accounts that only require a Baby JubJub key. These accounts are called `internal accounts`.
+
+The advantage of these accounts is that are much cheaper to create than a `normal account`, since these accounts only exist on Hermez. The downside is that one cannot perform deposits or withdrawals from this type of accounts. However, there are some scenarios where these accounts are useful. For example, in those scenarios where one requires a temporary account. (for example, Exchanges could use these accounts to receive a transfer from users).
```

Comment:
```suggestion
The advantage of these accounts is that they are much more inexpensive to create than a `normal account` since these accounts only exist on Hermez. The downside is that one cannot perform deposits or withdrawals from this type of account. However, there are some scenarios where these accounts are useful. For example, in those scenarios where one requires a temporary account. (for example, Exchanges could use these accounts to receive a transfer from users).
```